bubblewrap.c

```diff
@@ -518,7 +518,7 @@ switch_to_user_with_privs (void)
   if (prctl (PR_SET_KEEPCAPS, 1, 0, 0, 0) < 0)
     die_with_error ("prctl(PR_SET_KEEPCAPS) failed");
 
-  if (setuid (real_uid) < 0)
+  if (setuid (opt_sandbox_uid) < 0)
```

Comment:
Hm, we're not currently validating that `opt_sandbox_uid` being set requires `--unshare-user` AFAICS.  Which I think means this could be used to gain uid 0 in the host userns, right?